doc/dadbod-ui.txt

```diff
@@ -714,6 +714,13 @@ g:db_ui_force_echo_notifications
 
 		Default value: `0`
 
+					     *g:db_ui_use_postgres_views*
+g:db_ui_use_postgres_views
+    Postgres database shows tables within schemas nicely. To show
+    tables nicely within Redshift set this value to 0.
```

Comment:
Let's rephrase this a bit, since postgres users can also opt out from views.

```suggestion
    Toggle showing postgres views in the drawer.
    This option must be disabled (set to 0) for Redshift.
```

Reply:
Copy and pasted your phrasing. :)

Reply:
Oh, wait I should change the readme as well to match your phrasing.

Reply:
done